src/earn/EarnConfirmationScreen.test.tsx

```diff
@@ -187,15 +183,6 @@ describe('EarnConfirmationScreen', () => {
     expect(getByTestId('EarnConfirmation/GasFeeFiatAmount')).toHaveTextContent('₱119.70')
     expect(queryByTestId('EarnConfirmation/GasSubsidized')).toBeFalsy()
     expect(getByTestId('EarnConfirmationScreen/CTA')).toBeEnabled()
-    // TODO(act-1389): update this test to make sure that reward positions are not included in partial withdrawals.
```

Comment:
Can you add this check the AC refers to?

Reply:
Small one added in 502a676148bbc2a0e3c217baa195907330415dd2.